kubech

```diff
@@ -54,7 +54,7 @@ chmod 744 "${KUBECONFIG_SRC_DIR}" "${KUBECONFIG_DEST_DIR}"
 # Handeling Kube config files.
 _kubeconfig_files () {
     if [ -d "${KUBECONFIG_SRC_DIR}" ]; then
-        KUBECONFIG_FILES="$(find "${KUBECONFIG_SRC_DIR}" -type f -printf '%p:')"
+        KUBECONFIG_FILES="$(find "${KUBECONFIG_SRC_DIR}" -type f -print0 | tr '/n' ':')"
```

Comment:
It should be `-print | tr '\n' ':'`

Reply:
His branch as-is works for me on macOS 12 using `-print0`.  But i did notice `"${KUBECONFIG_SRC_DIR}` looks at `~/.kube/config.src.d/` for me which is empty.

Reply:
@defenestration that's why it doesn't really work (it will put everything as 1 line and there is no new line in the output).
Could you please test it using `-print | tr '\n' ':'`?

Reply:
Seems to work fine that way too, here is the diff against the branch in the PR for confirmation

```diff
diff --git a/kubech b/kubech
index 5576f70..68c60e9 100644
--- a/kubech
+++ b/kubech
@@ -54,7 +54,7 @@ chmod 744 "${KUBECONFIG_SRC_DIR}" "${KUBECONFIG_DEST_DIR}"
 # Handeling Kube config files.
 _kubeconfig_files () {
     if [ -d "${KUBECONFIG_SRC_DIR}" ]; then
-        KUBECONFIG_FILES="$(find "${KUBECONFIG_SRC_DIR}" -type f -print0 | tr '/n' ':')"
+        KUBECONFIG_FILES="$(find "${KUBECONFIG_SRC_DIR}" -type f -print | tr '\n' ':')"
     fi
     echo "${KUBECONFIG_FILES}${KUBECONFIG_ORIG}"
 }
```

Reply:
@defenestration thanks a lot for your feedback :ok_hand: